uniffi/src/lib.rs

```diff
@@ -27,7 +27,7 @@
 //! in a `.idl` file, where it can be processed by the tools from this crate.
 //! For example you might define an interface like this:
 //!
-//! ```
+//! ```text
```

Comment:
Otherwise `cargo test` tries to run these as doctests by default, which doesn't end well.